README.md

```diff
@@ -79,7 +77,7 @@ in your program simply via:
 >>> IBMQ.load_account()
 ```
 
-Those who do not want to save their credentials to disk should use instead:
+Those who do not want to save their credentials in a persistent file should use instead:
 
 ```python
 >>> from qiskit import IBMQ
```

Comment:
Should we really have this in the qiskit-terra readme? Especially with the state of flux around ibm provider packages maybe we should just link to the readme of the provider package du jour and as we shouldn't want to worry about how to setup authentication or credentials as part of qiskit terra.

Reply:
I agree. However, let's keep it for now until the provider README gets released and updated.